src/qr.nim

```diff
@@ -75,36 +74,27 @@ func qrRow*(data: string, eccLevel = Ecc_Medium, verMin: cint = VERSION_MIN, ver
         output.add(0)
       inc(x)
     inc(y)
-    container.add(output)
+    result.add output
     output = @[]
-  return container
 
 
-proc qrSvgGenerate(qrcodeData: ptr uint8, svgSize: int32, border: cint, fgColor = "#000000", bgColor = "#ffffff"): string =
+func qrSvgGenerate(qrcodeData: ptr uint8, svgSize: int32, border: cint, fgColor: static string = "#000000", bgColor: static string = "#ffffff"): string {.noinit.} =
```

Comment:
Is the `static string` just the type definition of `bgColor`?

`bgColor: static string = "#ffffff"`

Reply:
Yes, only for `bgColor` and `fgColor`.